### PR TITLE
Setup automated release system.

### PR DESCRIPTION
**Pull request by rwjblue**
This the result of running `npx create-rwjblue-release-it-setup --update`.

After this change, we should change our process to:

* Ensure that each pull request that is merged has the appropriate label (bug, enhancement, internal, documentation, etc).
* Update pull request titles to be a good changelog entry (tweaking the verbiage and whatnot to be more "customer" oriented)
* Ensure that the prerequisites listed in `RELEASE.md` are satisified (primarily a `GITHUB_AUTH` environment variable)
* Use `npx release-it` for the actual release process itself

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,47 +1,62 @@
...
-Clean up the generated changelog by inserting the appropriate verion number and removing merge commits and previous release commits if necessary.
+When reviewing merged PR's the labels to be used are:
 
-Review the changes and then commit them with a message like:
+* breaking - Used when the PR is considered a breaking change.
+* enhancement - Used when the PR adds a new feature or enhancement.
+* bug - Used when the PR fixes a bug included in a previous release.
+* documentation - Used when the PR adds or updates documentation.
+* internal - Used for internal changes that still require a mention in the
+  changelog/release notes.
```
Comment (drewlee) [inline RELEASE.md:26]: Just wondering, what would a minor/patch dependency update fall under, enhancement?

Comment (MelSumner) [reply RELEASE.md:26]: @drewlee wouldn't that be considered internal?

Comment (drewlee) [reply RELEASE.md:26]: Yeah, that makes sense @MelSumner. Thanks for clarifying.

File: RELEASE.md [hunk 2]
```diff
@@ -1,47 +1,62 @@
...
 
-```bash
-git commit -am "Update CHANGELOG for x.x.x"
-```
 
-Where `x.x.x` is the version you are releasing.
+## Release
 
-## Bump The Version
+Once the prep work is completed, the actual release is straight forward:
```
Comment (MelSumner) [inline RELEASE.md]:
```suggestion
Once the prep work is completed, the release itself is straightforward:
```

Comment (rwjblue) [reply RELEASE.md]: Ohh, thank you for this! This file is itself automatically generated from https://github.com/rwjblue/create-rwjblue-release-it-setup/blob/master/release-template.md.

@MelSumner - Would you mind sending off a PR over there too?

**Pull request merged**